src/connection_string.ts

```diff
@@ -360,6 +375,18 @@ export function parseOptions(
     );
   }
 
+  if (
```

Comment:
can this check be in the else clause of the `if (isSRV)` check above that? would be nice to have srv-related stuff grouped together, and this isn't relying on the processed option values anyway